raiden/tests/unit/rif_comms/test_rif_comms_client.py

```diff
@@ -1,181 +1,145 @@
-import json
-import secrets
-import time
 import unittest
 
-import grpc
 import pytest
-from coincurve import PublicKey
-from eth_utils import to_checksum_address, to_canonical_address
-from sha3 import keccak_256
+from eth_utils import to_canonical_address
 from transport.rif_comms.client import Client as RIFCommsClient
-from transport.rif_comms.proto.api_pb2 import Channel, PublishPayload, Msg
-from transport.rif_comms.proto.api_pb2_grpc import CommunicationsApiStub
-
-
-def get_random_address_str() -> str:
-    private_key = keccak_256(secrets.token_bytes(32)).digest()
-    public_key = PublicKey.from_valid_secret(private_key).format(compressed=False)[1:]
-    addr = keccak_256(public_key).digest()[-20:]
-    return to_checksum_address(addr)
-
-
-LUMINO_1_ADDRESS = to_canonical_address("0x8cb891510dF75C223C53f910A98c3b61B9083c3B")
-LUMINO_1_COMMS_API = "localhost:5013"
-
-LUMINO_2_COMMS_API = "localhost:5016"
-LUMINO_2_ADDRESS = to_canonical_address("0xeBfF0EEe8E2b6952E589B0475e3F0E34dA0655B1")
-
-LUMINO_3_ADDRESS = "0x636BA79E46E0594ECbbEBb4F74B9336Fd4454442"
+from transport.rif_comms.utils import notification_to_payload
+
+test_nodes = {
+    1: {
+        "address": to_canonical_address("0x8cb891510dF75C223C53f910A98c3b61B9083c3B"),
+        "comms-api": "localhost:5013",
+    },
+    2: {
+        "address": to_canonical_address("0xeBfF0EEe8E2b6952E589B0475e3F0E34dA0655B1"),
+        "comms-api": "localhost:6013",
+    },
+    3: {
+        "address": to_canonical_address("0x636BA79E46E0594ECbbEBb4F74B9336Fd4454442"),
+        "comms-api": "localhost:7013",
+    },
+}
 
 
 @pytest.mark.usefixtures("rif_comms_client")
 @pytest.fixture(scope="class")
 def rif_comms_client(request):
```

Comment:
i don't particularly like the way this fixture is written, but changing it is out of scope for now